src/features/SwapAssets/stateFormFactory.tsx

```diff
@@ -34,51 +34,24 @@ const swapFormInitialState = {
   lastChangedAmount: LAST_CHANGED_AMOUNT.FROM
 };
 
-const BASE_ASSET_ADDRESS = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee';
-
 const SwapFormFactory: TUseStateReducerFactory<SwapFormState> = ({ state, setState }) => {
   const network = useSelector(selectNetwork(state.selectedNetwork)) as Network;
+  const baseAsset = useSelector(getBaseAssetByNetwork(network));
+  const assets = useSelector(getSwapAssetsByNetwork(state.selectedNetwork));
+  const sortedAssets = assets.sort((asset1, asset2) => asset1.ticker.localeCompare(asset2.ticker));
 
-  const setNetwork = (network: NetworkId) => {
+  useEffect(() => {
     setState((prevState: SwapFormState) => ({
       ...prevState,
-      selectedNetwork: network
+      fromAsset: baseAsset,
+      toAsset: sortedAssets.filter((a) => a.uuid !== baseAsset.uuid)[0]
```

Comment:
Can you explain this?

Reply:
We preselect `baseAsset` as the fromAsset. Then for the toAsset, we preselect the first value from the asset list that isn't the `baseAsset`, since we can't swap between two assets that are the same. It's just a way to preselect some assets before the user starts selecting things.